src/adapters/notifications/NotificationCenterAdapter.ts

```diff
@@ -97,7 +99,7 @@ export class NotificationCenterAdapter implements NotificationAdapterInterface {
       segment: {
         analyticsUserId: user.segmentUserId(),
         anonymousId: user.segmentUserId(),
-        payload: getSegmentDonationAttributes({
```

Comment:
Please change name of this function to something like generateEmailPayload , .. we dont want to use Segment name anymroe

Reply:
Done!